### ISSUE TITLE
Menu Bar adds to window content height causing scrollbar

### ISSUE THREAD

**Issue by Puxap-3opre** [scanline/neutralinojs-menubar#1]
Using menubar adds extra space to window contents height causing the vertical scrollbar to appear, even if the window content is small and no scrollbar would be shown normally. I tried to play with `margin-top`, `margin-bottom` etc., but couldn't correct this.
![menubar_space](https://github.com/user-attachments/assets/409fd0a8-e093-446f-ae5e-7615602bea08)

**Comment by scanline** [comment]
From the partly visible application bar I assume you're running it on a Mac, are you? I didn't test this target as of yet simply because this library aims at Windows & Linux.
I know how to fix it though.

**Comment by Puxap-3opre** [comment]
> From the partly visible application bar I assume you're running it on a Mac, are you? I didn't test this target as of yet simply because this library aims at Windows & Linux. I know how to fix it though.

No, it's Linux with some WM theme from https://www.xfce-look.org/browse?cat=138&ord=rating

Thank you very much!

I like NeutralinoJS, and a nice looking MenuBar is very useful to have.

**Comment by scanline** [comment]
Ah it's Linux! It's interesting, the last time I've tested this on Linux it was working. This time the scrollbar also appeared over here.
Can you try changing a CSS property from the `<menu-container>` element via the inspector and tell me if it fixes the issue?
`height: -webkit-fill-available;`
to
`height: calc(100vh - 27px);`

By the way, nice to hear that the library is useful to you. Thanks!

**Comment by Puxap-3opre** [comment]
> Can you try changing a CSS property from the `<menu-container>` element via the inspector and tell me if it fixes the issue?
> `height: calc(100vh - 27px);`

Yes, it works. It seems that this (in my css file) works as well:
```
menu-container {
    height: auto !important;
}
```

TY!
